.kokoro/build.sh

```diff
@@ -69,14 +69,22 @@ integration)
     RETURN_CODE=$?
     ;;
 samples)
-    if [[ -f samples/pom.xml ]]
+    SAMPLES_DIR=samples
+    if [[ -f SAMPLES_DIR/pom.xml ]]
     then
         if [ -f "${KOKORO_GFILE_DIR}/secret_manager/java-bigquerydatatransfer-samples-secrets" ]
         then
             source "${KOKORO_GFILE_DIR}/secret_manager/java-bigquerydatatransfer-samples-secrets"
         fi
 
-        pushd samples
+        # only run ITs in snapshot/ on presubmit PRs
+        if [[ ! -z ${KOKORO_GITHUB_PULL_REQUEST_NUMBER} ]]
+        then
+          SAMPLES_DIR=samples/snapshot
+        fi
```

Comment:
Let's move this up by the first declaration of SAMPLES_DIR

Reply:
done